diff -up zabbix-5.4.11/src/libs/zbxicmpping/icmpping.c.orig zabbix-5.4.11/src/libs/zbxicmpping/icmpping.c
--- zabbix-5.4.11/src/libs/zbxicmpping/icmpping.c.orig	2022-03-18 13:18:46.003837766 +0300
+++ zabbix-5.4.11/src/libs/zbxicmpping/icmpping.c	2022-03-18 13:20:30.856464116 +0300
@@ -54,34 +54,7 @@ static time_t	fping_check_reset_at;	/* t
 
 static void	get_source_ip_option(const char *fping, const char **option, unsigned char *checked)
 {
-	FILE	*f;
-	char	*p, tmp[MAX_STRING_LEN];
-
-	zbx_snprintf(tmp, sizeof(tmp), "%s -h 2>&1", fping);
-
-	if (NULL == (f = popen(tmp, "r")))
-		return;
-
-	while (NULL != zbx_fgets(tmp, sizeof(tmp), f))
-	{
-		for (p = tmp; isspace(*p); p++)
-			;
-
-		if ('-' == p[0] && 'I' == p[1] && (isspace(p[2]) || ',' == p[2]))
-		{
-			*option = "-I";
-			continue;
-		}
-
-		if ('-' == p[0] && 'S' == p[1] && (isspace(p[2]) || ',' == p[2]))
-		{
-			*option = "-S";
-			break;
-		}
-	}
-
-	pclose(f);
-
+	*option = "-S";
 	*checked = 1;
 }
 
